katas/content/superdense_coding/entangled_pair/PlaceHolder.qs

```diff
@@ -0,0 +1,5 @@
+namespace Kata {
```

Comment:
The filename can be case sensitive (Linux), and it should be Placeholder.qs (lowercase h)